PROJECT

```diff
@@ -13,6 +13,14 @@ resources:
   kind: ServiceBinding
   path: github.com/servicebinding/service-binding-controller/apis/v1alpha3
   version: v1alpha3
+- api:
```

Comment:
I made these edits by hand. I'm not exactly sure what this should look like for projects with multiple versions

Reply:
I think it looks good. Here is an example used in the [doc](https://book.kubebuilder.io/multiversion-tutorial/tutorial.html): https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/multiversion-tutorial/testdata/project/PROJECT